app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -1133,6 +1133,28 @@ protected void setupCard(
         }
     }
 
+    public void showStatusContent(boolean show) {
+        int visibility = show ? View.VISIBLE : View.GONE;
+        avatar.setVisibility(visibility);
+        avatarInset.setVisibility(visibility);
+        displayName.setVisibility(visibility);
+        username.setVisibility(visibility);
+        timestampInfo.setVisibility(visibility);
+        contentWarningDescription.setVisibility(visibility);
+        contentWarningButton.setVisibility(visibility);
+        content.setVisibility(visibility);
+        cardView.setVisibility(visibility);
+        itemView.findViewById(R.id.status_media_preview_container).setVisibility(visibility);
```

Comment:
uhm wouldn't `findViewById()` be *super* slow, especially for each element?

Reply:
Not super slow, but it is indeed not ideal. Will change.